### ISSUE TITLE
Error: Unable to access jarfile current-wind-surface-level-gfs-1.0.json       

### ISSUE THREAD

**Issue by techscopio** [cambecc/earth#104]
When I am trying to run the command **grib2json -d -n -o current-wind-surface-level-gfs-1.0.json gfs.t00z.pgrb2.1p00.f000**  , it shows like "**Error: Unable to access jarfile current-wind-surface-level-gfs-1.0.json       "** Please help

**Comment by rinku16** [comment]
yes you need to manually download it by going throgh web page (https://nomads.ncep.noaa.gov/cgi-bin/filter_gfs_1p00.pl), this is my blind arrow.
This Project lacks correct link references in commands.
This Project does not have script which i think download the current temperature and climate information form US weather service. If you need any help plz message me i am also working over this project.
